html/elements/input/color.json

```diff
@@ -21,7 +21,8 @@
                 "notes": "Firefox doesn't yet support inputs of type <code>color</code> on Windows Touch."
```

Comment:
Do we have any clue what Windows Touch means? I can tap the element through touchscreen to open the color picker.

Reply:
Windoes Touch I think is Windows with a touch screen, like the 2-in-1s

Reply:
In that case we should also reevaluate this as Firefox on Windows doesn't do anything different on touchscreen. It might also mean the old Windows 8 Metro-style Firefox.